docs/SRS-Volere/SRS.tex

```diff
@@ -170,15 +170,32 @@ \subsection{Longevity Requirements}
 
 \section{Operational and Environmental Requirements}
 \subsection{Expected Physical Environment}
-\lips
+\begin{itemize}
+    \item The application is expected to be compatible with smart mobile devices running Android.
+    \item The application is expected to be compatible with smart mobile devices running IOS.
+    \item The application is expected to be compatible with tablet devices running iPadOS or Android. 
+\end{itemize}
 \subsection{Wider Environment Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{WER1}: The application is designed to operate on devices equipped with camera peripherals.
+    \item \textbf{WER2}: The application is designed to operate on devices that have access to the internet. 
+\end{itemize}
 \subsection{Requirements for Interfacing with Adjacent Systems}
-\lips
+\begin{itemize}
+    \item \textbf{IASR1}: The application shall work with iOS 16.
+    \item \textbf{IASR2}: The application shall work with Android 13.
+\end{itemize}
 \subsection{Productization Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{PRR1}: The application shall include a detailed online help documentation accessible from within the application.
+    \item \textbf{PRR2}: The application shall provide a quick and user-friendly guide/tour to familiarise user with application functions.
+\end{itemize}
 \subsection{Release Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{RER1}: The application shall undergo thorough testing and quality control procedures prior to each application releases and updates.
```

Comment:
I'm starting to think maybe thorough testing may be too vague/not verifiable? Maybe add some sort of coverage metric such as 95%? Not sure how achievable that is but it gives us a metric.

Reply:
resolved. added code coverage metric.